src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfUploadService.java

```diff
@@ -133,14 +139,15 @@ public List<PdfDecodedMultipartFile> convertCaseDataToPdfDecodedMultipartFile(Ca
         }
         try {
             if (WELSH_LANGUAGE.equals(GenericServiceUtil.findClaimantLanguage(caseData))) {
-                byte[] pdfData = pdfService.convertCaseToPdf(caseData, this.welshPdfTemplateSource);
+                byte[] pdfData = pdfService.convertCaseToPdf(caseData, this.welshPdfTemplateSource, PDF_TYPE_ET1,
+                                                             ET3_FORM_CLIENT_TYPE_RESPONDENT, SUBMIT_ET3_CITIZEN);
```

Comment:
If this is for the ET1, why are we referencing ET3 and the Respondent?

Reply:
New parameter CLIENT_TYPE_RESPONDENT not has any effect to the flow of ET1 pdf creation. It is used to discriminate representatives and respondents while mapping ET3 PDF data. SUBMIT_ET1_CITIZEN does not have any effect to the flow of ET1 pdf creation just checks if the event is submission event not. If submit event, sets date received field of pdf file to the local date current value. This is because, on caseworker screens we create PDF files without submission of the event. To not to cause any misunderstanding and conflict I updated names of the constants as CLIENT_TYPE_RESPONDENT and SUBMIT_ET1_CITIZEN.

Reply:
Updated as
byte[] pdfData = pdfService.convertCaseToPdf(caseData, this.welshPdfTemplateSource, PDF_TYPE_ET1,
                                                             CLIENT_TYPE_CLAIMANT, SUBMIT_ET1_CITIZEN);